graphql-java-support/src/main/resources/definitions_fed2_5.graphqls

```diff
@@ -0,0 +1,83 @@
+#
+# https://specs.apollo.dev/federation/v2.0/federation-v2.0.graphql
+#
+
+directive @key(fields: FieldSet!, resolvable: Boolean = true) repeatable on OBJECT | INTERFACE
+directive @requires(fields: FieldSet!) on FIELD_DEFINITION
+directive @provides(fields: FieldSet!) on FIELD_DEFINITION
+directive @external on OBJECT | FIELD_DEFINITION
+directive @extends on OBJECT | INTERFACE
+directive @override(from: String!) on FIELD_DEFINITION
+directive @inaccessible on
+    | FIELD_DEFINITION
+    | OBJECT
+    | INTERFACE
+    | UNION
+    | ENUM
+    | ENUM_VALUE
+    | SCALAR
+    | INPUT_OBJECT
+    | INPUT_FIELD_DEFINITION
+    | ARGUMENT_DEFINITION
+directive @tag(name: String!) repeatable on
+    | FIELD_DEFINITION
+    | INTERFACE
+    | OBJECT
+    | UNION
+    | ARGUMENT_DEFINITION
+    | SCALAR
+    | ENUM
+    | ENUM_VALUE
+    | INPUT_OBJECT
+    | INPUT_FIELD_DEFINITION
+scalar FieldSet
+
+#
+# https://specs.apollo.dev/link/v1.0/link-v1.0.graphql
+#
+
+directive @link(
+    url: String!,
+    as: String,
+    import: [Import])
+repeatable on SCHEMA
+
+scalar Import
+
+#
+# federation-v2.1
+#
+
+directive @composeDirective(name: String!) repeatable on SCHEMA
+
+#
+# federation-v2.2
+#
+
+directive @shareable repeatable on FIELD_DEFINITION | OBJECT
+
+#
+# federation-v2.3
+#
+
+directive @interfaceObject on OBJECT
+
+#
+# federation-v2.5
+#
+
+directive @authenticated on
+    ENUM
+  | FIELD_DEFINITION
+  | INTERFACE
+  | OBJECT
+  | SCALAR
+
+directive @requiresScopes(scopes: [[Scope!]!]!) on
+    ENUM
+  | FIELD_DEFINITION
+  | INTERFACE
+  | OBJECT
+  | SCALAR
+
+scalar Scope
```

Comment:
Pretty sure this needs to be `federation__Scope` everywhere it's defined and referenced by `@requiresScopes`. I would expect a composition error / incompatible definition for this.

```
Invalid definition for directive "@requiresScopes": argument "scopes" should have type "[[federation__Scope!]!]!" but found type "[[Scope!]!]!"
```

Reply:
`federation__` namespace prefix should only be required if given directive/type is not explicitly imported (see the diff between the [custom authenticated directive with no imports](https://github.com/apollographql/federation-jvm/blob/379008e354d21203da1a33a8d73c8e339a547035/graphql-java-support/src/test/resources/schemas/customAuthenticated_federated.graphql#L23) and [default authorization with imports](https://github.com/apollographql/federation-jvm/blob/379008e354d21203da1a33a8d73c8e339a547035/graphql-java-support/src/test/resources/schemas/authorization_federated.graphql#L29)